Cargo.toml

```diff
@@ -3,6 +3,23 @@ name = "vhost"
 version = "0.1.0"
 authors = ["Liu Jiang <gerry@linux.alibaba.com>"]
 repository = "https://github.com/rust-vmm/vhost"
-license = "Apache-2.0"
+license = "Apache-2.0 or BSD-3-Clause"
+
+[features]
+default = []
+vhost-vsock = []
+vhost-kern = ["vm-memory"]
+vhost-user-master = []
+vhost-user-slave = []
 
 [dependencies]
+bitflags = "1.1.0"
+libc = "0.2.67"
+vmm-sys-util = ">=0.3.1"
+
+[dependencies.vm-memory]
+git = "https://github.com/rust-vmm/vm-memory"
+optional = true
+
+[dev-dependencies]
+tempfile = "3.1.0"
```

Comment:
You don't need this dependency as `tempfile` abstractions are already defined in vmm-sys-util.

Reply:
Yes, we also plan to remove the depdency for tempfile